scripts/centrifuge/run_chain_ci.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+S3_URL="https://centchain.nyc3.digitaloceanspaces.com"
```

Comment:
How does this s3 bucket get updated?

Reply:
Pushed from travis in our centrifuge-chain repo